frontend/src/config/translations.ts

```diff
@@ -15,7 +15,8 @@ export type TranslationEntryKey =
   | "page.Groups.types.Administrative"
   | "page.Groups.types.Community"
   | "page.Groups.types.Education"
-  | "page.Groups.types.Activities";
+  | "page.Groups.types.Activities"
```

Comment:
should we put ; in this line?

Reply:
No. One more item is added in line 19. Therefore `;` which signifies the end of the items should be at the end of line 19.

Reply:
ohhh got it, thank you!